crates/rpc/src/lib.rs

```diff
@@ -129,7 +129,11 @@ impl Handler {
 
     async fn accounts(_: Params, _: Ctx) -> jsonrpc_core::Result<serde_json::Value> {
         let wallets = Wallets::read().await;
-        let address = wallets.get_current_wallet().get_current_address().await;
+        let address = wallets
+            .get_current_wallet()
+            .ok_or(Error::Wallets(iron_wallets::Error::NoWallet))?
```

Comment:
I believe this should work, no?

```suggestion
            .ok_or(iron_wallets::Error::NoWallet)?
```

Reply:
Added this on the rpc crate:
```
#[error(transparent)]
Wallets(#[from] iron_wallets::Error)
```
`.ok_or(iron_wallets::Error::NoWallet)?` did not work
Tried other ways to make it less verbose but none of them compiled